src/cosalib/ibmcloud.py

```diff
@@ -99,6 +97,7 @@ def write_ova(self, image_name):
         log.debug(ovf_xml)
         # OVF descriptor must come first, then the manifest, then the meta file
         self.tar_members.append(self.ovf_path)
+        self.tar_members.append(f'{ovf_params["image"]}-disk.raw')
```

Comment:
would it be bad to drop the `-disk` part? 

```suggestion
        self.tar_members.append(f'{ovf_params["image"]}.raw')
```

I guess it really doesn't matter?

Reply:
I had the same thought but it's hard to tell without knowing in what context the string shows up. Though it at least bears some resemblance to other OVAs we produce.

So, dunno. I'll re-enable auto-merge and if someone thinks it'd look better in the UI without it, it can be done in a follow-up.